src/google/protobuf/generated_message_tctable_gen.cc

```diff
@@ -753,34 +464,6 @@ TailCallTableInfo::TailCallTableInfo(
       : message_options.is_lite ? TcParseFunction::kGenericFallbackLite
                                 : TcParseFunction::kGenericFallback;
 
-  if (descriptor->options().message_set_wire_format()) {
-    ABSL_DCHECK(ordered_fields.empty());
-    ABSL_DCHECK(inlined_string_indices.empty());
-    if (message_options.uses_codegen) {
-      fast_path_fields = {{TailCallTableInfo::FastFieldInfo::NonField{
-          message_options.is_lite
-              ? TcParseFunction::kMessageSetWireFormatParseLoopLite
-              : TcParseFunction::kMessageSetWireFormatParseLoop,
-          0, 0}}};
-
-      aux_entries = {{kSelfVerifyFunc}};
```

Comment:
Removing this is problematic as we internally need verify function in aux table.